lib/pipekit/deal.rb

```diff
@@ -2,8 +2,9 @@ module Pipekit
   class Deal
     include Repository
 
-    def get_by_person_id(person_id)
-      request.get("/#{Pipekit::Person.uri}/#{person_id}/#{self.class.uri}")
+    def get_by_person_id(person_id, request = nil)
+      request ||= Request.new(Person.resource)
```

Comment:
Seems weird that `Request` is being injected here - would calling code ever pass in a Request object or is it just for testing purposes?